model/trace/database.yaml

```diff
@@ -196,33 +224,37 @@ groups:
 
   - id: db.mongodb
     type: span
-    extends: attributes.db.client.minimal
+    extends: trace.db.common.minimal
     brief: >
       Attributes for MongoDB
     attributes:
       - ref: db.operation.name
+        sampling_relevant: true
         brief: >
           The name of the command being executed.
         note: >
           See [MongoDB database commands](https://www.mongodb.com/docs/manual/reference/command/).
         examples: ['findAndModify', 'getMore', 'update']
       - ref: db.collection.name
+        sampling_relevant: true
         brief:
           The MongoDB collection being accessed within the database stated in `db.namespace`.
         requirement_level: required
       - ref: db.namespace
+        sampling_relevant: true
         brief: The MongoDB database name.
         requirement_level:
           conditionally_required: If available.
         note: ""  # overriding the base note
 
   - id: db.elasticsearch
```

Comment:
For Elasticsearch, we also have `db.elasticsearch.path_parts.<key>` available during span creating, so potentially we could also consider this for sampling relevant. I'm not sure we want to have these db specific fields, or just common ones.